server/game/cards/locations/03/towerofthehand.js

```diff
@@ -33,6 +33,8 @@ class TowerOfTheHand extends DrawCard {
 
     onOpponentCardSelected(player, card) {
         card.controller.returnCardToHand(card);
+		this.selectedCard.controller.returnCardToHand(this.selectedCard);
```

Comment:
Good catch.  There seems to be a spacing issue here though, can you check?  Also can you update the text below to say '{0} kneels {1}' rather than uses